src/resolve/resolved-dns-cache.c

```diff
@@ -628,7 +628,8 @@ int dns_cache_put(
                 uint32_t nsec_ttl,
                 usec_t timestamp,
                 int owner_family,
-                const union in_addr_union *owner_address) {
+                const union in_addr_union *owner_address,
+                DnsCacheMode cache_mode) {
```

Comment:
It doesn't matter a great deal, but I'd prefer to have this as the second argument (right after `DnsCache*`). It seems to fit conceptually with the cache more than with the answer.

Reply:
Done, I agree with your observation.